HyperV/scripts/create-environment.ps1

```diff
@@ -211,15 +212,19 @@ function cherry_pick($commit) {
 }
 
 ExecRetry {
+    ls C:\OpenStack\build\openstack\neutron
     pushd C:\OpenStack\build\openstack\neutron
```

Comment:
if we do this anyway, why don't we use the variables defined instead of hardcoded paths everywhere?